app/src/main/java/net/opendasharchive/openarchive/features/main/MainActivity.kt

```diff
@@ -285,6 +279,16 @@ class MainActivity : BaseActivity(), FolderAdapterListener, SpaceAdapterListener
         }
     }
 
+    private fun updateCurrentSpaceAtDrawer() {
+        mBinding.currentSpaceName.text = Space.current?.friendlyName
+        mBinding.currentSpaceName.setDrawable(
+            Space.current?.getAvatar(applicationContext)?.scaled(32, applicationContext),
```

Comment:
This 32 should come from dimensions, isn't it?
Why are we sending application context?

Reply:
ApplicationContext is used in getAvatar function to retrieve Resource Colors
```kotlin
val color = ContextCompat.getColor(context, R.color.colorOnBackground)
```